subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformValuesInjectionIntegrationTest.groovy

```diff
@@ -142,6 +145,7 @@ class ArtifactTransformValuesInjectionIntegrationTest extends AbstractDependency
         failure.assertHasCause("Property 'extension' is not annotated with an input annotation.")
         failure.assertHasCause("Property 'outputDir' is annotated with unsupported annotation @OutputDirectory.")
         failure.assertHasCause("Property 'missingInput' does not have a value specified.")
+        failure.assertHasCause("Property 'inputFiles' must not have absolute path sensitivity declared. Use a different normalization for cacheable transform inputs.")
```

Comment:
This error message is a bit misleading. Absolute path sensitivity wasn't declared in this case, but instead the problem is that a sensitivity other than absolute wasn't declared.

Reply:
We now show a different error depending on whether nothing was declared or something was declared.